lib/src/utils/commands_extension.dart

```diff
@@ -276,6 +277,14 @@ extension CommandsClientExtension on Client {
         txid: args.txid,
       );
     });
+    addCommand('sendRaw', (args) async {
```

Comment:
Wouldn't this cause a problem, if this command was available in TIM clients?

Reply:
TIM-Requirement is only to not be able to send custom state events afaik